src/Microsoft.DotNet.Interactive.CSharp/InteractiveWorkspace.cs

```diff
@@ -61,6 +62,24 @@ public void AddSubmission(ScriptState scriptState)
             var debugName = assemblyName;
             var projectId = ProjectId.CreateNewId(debugName: debugName);
 
+            solution = CreateProjectAndAddToSolution(projectId, debugName, assemblyName, compilationOptions, solution,
+                projectReferenceProjectId);
+
+            var workingDocumentId = DocumentId.CreateNewId(
+                projectId,
+                debugName: $"working document for {submission}");
```

Comment:
Why have a `debugName` variable and also this?

Reply:
because that is the debug name of the document, not of the project

Reply:
using variable only